jupyterlab_caip_optimizer/src/components/suggest_trials/loading.tsx

```diff
@@ -0,0 +1,36 @@
+/**
```

Comment:
Could we possibly use this in other components?

Reply:
Yeah! I think the `Dashboard` and `Sidebar` would need to be updated in that case.

Reply:
Should I do that in a separate PR?

Reply:
Separate PR sounds good to me